systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlApiST.java

```diff
@@ -52,81 +47,59 @@ void testCruiseControlBasicAPIRequests(ExtensionContext extensionContext)  {
         resourceManager.createResourceWithWait(extensionContext, KafkaTemplates.kafkaWithCruiseControl(testStorage.getClusterName(), 3, 3).build());
 
         LOGGER.info("----> CRUISE CONTROL DEPLOYMENT STATE ENDPOINT <----");
-
-        String response = CruiseControlUtils.callApi(testStorage.getNamespaceName(), CruiseControlUtils.SupportedHttpMethods.POST, CruiseControlEndpoints.STATE,  CruiseControlUtils.SupportedSchemes.HTTPS, true);
-
-        assertThat(response, is("Unrecognized endpoint in request '/state'\n" +
-            "Supported POST endpoints: [ADD_BROKER, REMOVE_BROKER, FIX_OFFLINE_REPLICAS, REBALANCE, STOP_PROPOSAL_EXECUTION, PAUSE_SAMPLING, " +
-                "RESUME_SAMPLING, DEMOTE_BROKER, ADMIN, REVIEW, TOPIC_CONFIGURATION, RIGHTSIZE, REMOVE_DISKS]\n"));
-
-        response = CruiseControlUtils.callApi(testStorage.getNamespaceName(), CruiseControlUtils.SupportedHttpMethods.GET, CruiseControlEndpoints.STATE,  CruiseControlUtils.SupportedSchemes.HTTPS, true);
-
         LOGGER.info("Verifying that {} REST API is available", CRUISE_CONTROL_NAME);
+        CruiseControlUtils.ApiResult response = CruiseControlUtils.callApi(testStorage.getNamespaceName(), CruiseControlUtils.HttpMethod.GET, CruiseControlEndpoints.STATE);
```

Comment:
So, what exactly does this whole test test? And why is it not suported in KRaft mode? I guess that question applies to more of the tests in this class.

Reply:
> So, what exactly does this whole test test?

This is just to test basic CC API functionality, that you can get the state, execute rebalances, stop proposal execution, get user task list containing the previously executed tasks

> And why is it not suported in KRaft mode?

Appears to be an oversight, let me clean that up

Reply:
Do we really need to test these? I would expect that those are:
* Tested by Cruise Controls own tests
* Used in the KafkaRebalance tests

So I would expect here more or less just some check that we _unlocked_ the REST API. But not detailed tests of different CC fetures.

Reply:
> So I would expect here more or less just some check that we unlocked the REST API. But not detailed tests of different CC fetures.

Yeah that sounds fair to me, I'll take a stab at it with the next commit